Userland/Applications/SystemMonitor/ProcessModel.cpp

```diff
@@ -442,7 +443,7 @@ static DeprecatedString read_command_line(pid_t pid)
 void ProcessModel::update()
 {
     auto previous_tid_count = m_threads.size();
-    auto all_processes = Core::ProcessStatisticsReader::get_all(m_proc_all);
+    auto all_processes = Core::ProcessStatisticsReader::get_all(true);
```

Comment:
Maybe instead of "fix blank usernames" we could say "and call ProcessStatisticsReader properly" or some such? And then mention in the commit description that the API of Core::ProcessStatisticsReader  changed since we started using it and no longer requires passing a file with the contents of /proc/all to it?